### PR TITLE
Content for beta assignment 4

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+* Access to Adobe Experience Platform
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup) (required to have a mobile property in Launch with Edge configuration setup).
+* Completion of [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events) (required to have a Product reviews schema and dataset).
+
+### Download the sample application
+
+**Note:** This step is not required if you already implemented the steps described in [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:17]: URL link for `Assignment 3 - XDM implementation` should be https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-4](https://github.com/adobe/aepsdk-sample-app-ios/archive/beta-assignment-4.zip).
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-4#installation).
+{% endtab %}
+{% endtabs %}
+
+### Set up the required fields
+
+**Note:** The steps in this section are not required if you already implemented the steps described in [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:41]: URL link for `Assignment 3 - XDM implementation` should be https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation

File: beta/experience-platform-extension/tutorials/README.md [hunk 3]
```diff
@@ -64,3 +64,20 @@ Get started with [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io
...
 * [ ] Do you prefer syncing the visitor identifiers using a specialized API (example [syncIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#syncidentifier)), XDM format, remote rules or a combination of these?
 
+### Assignment 4 (week Nov 10)
+
+In this assignment you will learn how to:
+
+1. Update an existing XDM Schema
+2. Create data elements and rules for mobile properties in Adobe Experience Launch
+3. Attach new data to XDM events
+4. Validate the rules are executed correctly with Project Griffon
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/README.md:74]: "... executed correctly with Assurance (formerly Project Griffon)"

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+### Create a mobile rule in Adobe Experience Launch
+
+1. Navigate to [Adobe Experience Launch](https://experience.adobe.com/launch) and select the mobile property you created in the previous assignments.
+
+2. Click on `Extensions` from the left panel and update the `Adobe Experience Platform Edge` extension to latest version.
+
+3. Click on `Data elements` from the left panel and the following data elements:
+
+   1. Create `App ID` data element:
+      - Click on the `Create New Data Element` button.
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:81]: "`Add Data Element`"

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+      - Select the `Mobile Core` Extension and `OS Version` Data Element Type.
+      - Set the name for this data element to `Operating system version` and click `Save`.
+   3. Create `Event timestamp` data element:
+      - Click on the `Add Data Element` button.
+      - Select the `Mobile Core` Extension and `Adobe Experience Platform Timestamp` Data Element Type.
+      - Set the name for this data element to `Event timestamp` and click `Save`.
+   
+4. Click on `Rules` from the left pannel and create the following rule:
+
+   - Click `Create New Rule`.
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:95]: "`Add Rule`"

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md (hunk 4 above)
Comment (emdobrin) [reply beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:81]: First time you create a data element it shows as `Create New Data Element`.

Comment (kevinlind) [reply beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:81]: Ah, ignore comment. "Create New Data Element" is correct when there are no existing data elements. :)

File: beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md (hunk 5 above)
Comment (kevinlind) [reply beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md:95]: Ah, ignore comment. "Create New Rule" is correct when there are no existing rules. :)

File: beta/experience-platform-extension/tutorials/README.md (hunk 3 above)
Comment (emdobrin) [reply beta/experience-platform-extension/tutorials/README.md]: Updated to `AEP Assurance (Project Griffon)`, we use both the names in our docs for now.

**Review by kevinlind — APPROVED** [review]

**Pull request merged**